telnetsrv/telnetsrvlib.py

```diff
@@ -1078,6 +1072,29 @@ def authentication_ok(self):
             # No authentication desired
             self.username = None
             return True
+        
+    def _get_cmd_method(self):
```

Comment:
I'm confused on the usage for this though.  Can't you just say `if len(self.COMMANDS.alias) > 1, False, else True?`

Reply:
why `.alias`?

Reply:
so you're trying to see if a command is an all_input_command vs the first input being the command followed by parameters, right?  If the logic for the command input sets `self.alias = []` (which it does on line 194), then can't you make the assumption that if length of alias is 0, it's an all_input_command?